lib/DimmableLightDevice.js

```diff
@@ -31,7 +40,7 @@ module.exports = {
 						this.log(err);
 					});
 
-				if (this.getCapabilityValue('onoff') === false) {
+				if (!this.getCapabilityValue('onoff')) {
```

Comment:
changed `"value" === false` to `!"value"` since `=== false` is very verbose for this if statement